dev-portal/src/components/SwaggerUiLayout.jsx

```diff
@@ -34,10 +34,12 @@ const InfoReplacement = ({ specSelectors }) => {
             <div style={{ marginRight: "20px" }}>
               <p style={{ fontWeight: "bold" }}>Version</p>
               <p style={{ fontWeight: "bold" }}>Endpoint</p>
+              {/* <p style={{ fontWeight: "bold" }}>Usage Plan</p> */}
```

Comment:
Are we planning on uncommenting these later? Or can we trash them?

Reply:
Potentially? I was thinking this would be used with #86.